src/main/java/com/github/perscholas/GradeEvaluator.java

```diff
@@ -5,26 +5,38 @@
  */
 public class GradeEvaluator {
     public Boolean isGradeA(Double score) {
-        return null;
+        return score <= 100 && score >= 90;
     }
-
+    
     public Boolean isGradeB(Double score) {
-        return null;
+        return score <= 89 && score >= 80;
     }
-
+    
     public Boolean isGradeC(Double score) {
-        return null;
+        return score <= 79 && score >= 70;
     }
-
+    
     public Boolean isGradeD(Double score) {
-        return null;
+        return score <= 69 && score >= 65;
     }
-
+    
     public Boolean isGradeF(Double score) {
-        return null;
+        return score <= 64;
     }
-
+    
     public Character getGrade(Double score) {
-        return null;
+        char grade = 'M';
+        if (isGradeA(score)) {
+            grade = 'A';
+        } else if (isGradeB(score)) {
+            grade = 'B';
+        } else if (isGradeC(score)) {
+            grade = 'C';
+        } else if (isGradeD(score)) {
+            grade = 'D';
+        } else if (isGradeF(score)) {//Check really isn't needed

```

Comment:
check is not necessary; also you can immediately return the value of `grade` upon deriving it... no need to store as a variable intermediately